migrations/postgres-integration/0001_initial.down.sql

```diff
@@ -0,0 +1,37 @@
+DROP INDEX IF EXISTS idx_device_up_received_at;

```

Comment:
This applies to both `.up.sql` and `.down.sql`:

Maybe it would be better if we don't create the indices using migrations, and leave this up to the end-user. Different use-cases require different indices. This amount of indices means writes are slower and more storage space is needed.

Reply:
okay, will remove all the indices from `.up.sql` and `.down.sql`